core/src/androidTest/java/org/hisp/dhis/android/core/category/CategoryCategoryComboLinkStoreShould.java

```diff
@@ -93,16 +161,45 @@ private void whenInsertNewCategoryCombo() {
     }
 
     private void whenInsertNewCategoryComboLink() {
```

Comment:
this method has two responsibilities, create a fixture (givens...) and realize actions (whens..), I think that the code would be more readable in tests where is used if split this method in two, one responsibility per method, can you refactor it?

Reply:
Sure!